Makefile

```
@@ -454,6 +454,13 @@ update-dependencies: ## Uses go get -u to update all the dependencies while hold
 	# Copy everything to vendor directory
 	$(GO) mod vendor
 
+vet: ## Run mattermost go vet specifi checks
+	$(GO) get -u github.com/mattermost/mattermost-govet
+	$(GO) vet -vettool=$(GOPATH)/bin/mattermost-govet -license $(TE_PACKAGES)
```

I think we can simplify this to
```suggestion
	$(GO) vet -vettool=$(GOPATH)/bin/mattermost-govet -license ./...
```

---

I think that is not simplifying, we are using variables to define the packages in team edition and packages in enterprise edition, I think is better to keep using that variables.

---

The selling point for using `./...` is that the output is much sorter.

Is there a reason to use `$TE_PACKAGES` besides consistency?